selftests/functional/test_replay_basic.py

```diff
@@ -58,9 +58,9 @@ def test_run_replay_latest(self):
         self.run_and_check(cmd_line, expected_rc)
 
     def test_run_replay_data(self):
-        file_list = ['multiplex', 'config', 'urls', 'pwd', 'args']
+        file_list = ['multiplex', 'config', 'urls', 'pwd', 'args', 'cmdline']
         for filename in file_list:
-            path = os.path.join(self.jobdir, 'replay', filename)
+            path = os.path.join(self.jobdir, 'jobdata', filename)
```

Comment:
One more thing, I don't see the results-dir documentation and as a matter of fact, it's not documented. Would you please include `jobdata` in the `ReferenceGuide`?